packages/web-components/fast-foundation/src/select/select.template.ts

```diff
@@ -46,17 +46,19 @@ export function selectTemplate<T extends FASTSelect>(
                         ${ref("control")}
                     >
                         ${startSlotTemplate(options)}
-                        <slot name="button-container">
-                            <div class="selected-value" part="selected-value">
-                                <slot name="selected-value">${x => x.displayValue}</slot>
-                            </div>
-                            <div aria-hidden="true" class="indicator" part="indicator">
-                                <slot name="indicator">
-                                    ${options.indicator ?? ""}
-                                </slot>
-                            </div>
-                        </slot>
+                        <div class="field" part="field">
+                            <slot name="field">${x => x.displayValue}</slot>
+                        </div>
                         ${endSlotTemplate(options)}
+                        <div
+                            class="open-close-icon"
+                            part="open-close-icon"
+                            aria-hidden="true"
+                        >
+                            <slot name="open-close-icon">
+                                ${options.openCloseIcon ?? ""}
+                            </slot>
```

Comment:
This appears to lose the ability to swap out the button container, which is a key feature - It seems like this is a much more significant change than perhaps was intended...

Reply:
Addressed this and aligned with Open UI.